### ISSUE TITLE
Possibility to show Exceptions Per Day

### ISSUE THREAD

**Issue by woehrl01** [exceptionless/Exceptionless.UI#45]
In the summary form there is the exceptions per hour number. 

Due to the great work of Exceptionless. We have some points where we don't have exceptions on many days. 

It would be great to be able to toggle between "exception per hour", "per day", "per week", "per month"

**Comment by niemyjski** [comment]
Yes, this could be triggered by the date range you are looking at. What do you think?

**Comment by niemyjski** [comment]
@woehrl01 do you think this should be based on the date range that's currently selected?
